### ISSUE TITLE
Accessibility: text and background color contrast not sufficient for 508 compliance

### ISSUE THREAD

**Issue by tsdower** [ivoa/vor-to-landing#1]
Running the ANDI 508 accessibility validator against the example page (http://dc.g-vo.org/LP/cds.vizier/j/apj/852/48) returns a few failures about insufficient contrast. Namely the gray and darkgray elements on white fail. This includes css class 'explanation' and dates and associated text under History. 

ANDI is a web accessibility validation tool tuned specifically for section 508 related website standards. While 508 references a set of technical requirements US federally funded projects should adhere to, it's a good bar to set regardless of project home. Free bookmarklet tool used for testing: https://www.ssa.gov/accessibility/andi/help/install.html

I can make a PR with css changes to make it pass. I'm sure there will be further tweaks to the template itself but I wanted to start with this low-hanging fruit to familiarize myself with the code.
